pmda/rdf.py

```diff
@@ -161,6 +161,21 @@ def _conclude(self, ):
         rdf = self.count / (density * vol * self.nf)
         self.rdf = rdf
 
+    def get_cdf(self):
+        """Calculate the cumulative distribution functions (CDF).
+        Note that this is the actual count within a given radius, i.e.,
+        :math:`N(r)`.
+
+        Returns
+        -------
+              cdf : numpy array
+                      a numpy array with the same structure as :attr:`rdf`
+        """
+        cdf = np.cumsum(self.count) / self.nf
+        self.cdf = cdf
```

Comment:
With a `get` method this should not be stored. That's confusing (the expectation for such a function is to return a result) and it also uses up memory.

If you want to implement it as a managed attribute you could do
```python
@property
def cdf(self):
    return np.cumsum(self.count) / self.nf
```
Or calculate the cdf in `_conclude`, remove `get_cdf`, and just access `InterRDF.cdf`.

Reply:
I prefer setting it as a property so that cdf will not be in memory when users don't want it.

Reply:
And I also changed the same function in InterRDF_s.

Reply:
... and in MDA!